package.json

```diff
@@ -11,7 +11,8 @@
     "build:dev": "webpack --watch",
     "start:dev": "nodemon server/index.js",
     "seed:db": "node database/seed.js",
-    "reset:db": "mysql -u root < database/schema.sql"
+    "reset:db": "mysql -u root < database/schema.sql",
+    "test": "jest --detectOpenHandles"
   },
   "dependencies": {
```

Comment:
It looks like you've misplaced some of your dependencies into the `devDependencies` object (react, react-dom, etc.)